client/src/locales/de.ts

```diff
@@ -5,30 +5,96 @@ export default {
 			description:
 				"Synchrone Wiedergabe in Echtzeit. Optionales Abstimmungssystem.\nDunkles Theme. Keine Anmeldung erforderlich. Alles Open Source.\nNoch nie war es so einfach, gemeinsam Videos anzuschauen.",
 			btns: {
+				create: "@:nav.create.title",
 				browse: "Räume durchsuchen",
 				source: "Quellcode ansehen",
 			},
 		},
+		intro: {
+			title: "Watch-Partys aus der Ferne sind jetzt viel einfacher",
+			name: "OpenTogetherTube",
+			text1: "ist eine Plattform zur Echtzeit-Videosynchronisation. Es ist einfach zu bedienen und erfordert keine Anmeldung. Erstelle einfach einen Raum, füge Videos hinzu und lade deine Freunde ein. Zack, fertig! Du bist nun startklar um mit deinen Freunden bis 3 Uhr nachts Videos zu gucken.",
```

Comment:
"...Zack, fertig!..." is a reference to this https://youtu.be/JEOD9cv9Qe8?t=73 (1:13-2:15)